### PR TITLE
Speed up ingestion using bulk insert / updates

### PR DESCRIPTION
**Pull request by jacobwegner**
Ingestion with `django-treebeard` is currently pretty slow, because there are _lot_ of INSERT and UPDATE statements happening.

I took a couple passes at the ingestion scripts today and have some promising early results; more to test and document, but I see the performance improvements below:

On `master`:

```
time python manage.py prepare_db
<snip>
--[Loading versions]--
Iliad: 15710 nodes.
Odyssey: 12132 nodes.
Crito: 483 nodes.
Enchiridion: 157 nodes.
The Epistle to Diognetus: 113 nodes.
The Epistle of Barnabas: 218 nodes.
The First Epistle of Clement: 469 nodes.
The Second Epistle of Clement: 139 nodes.
The Didache: 119 nodes.
The Shepherd of Hermas: 923 nodes.
Ignatius to the Ephesians: 71 nodes.
Ignatius to the Magnesians: 43 nodes.
Ignatius to the Trallians: 45 nodes.
Ignatius to the Romans: 41 nodes.
Ignatius to the Philadelphians: 37 nodes.
Ignatius to the Smyrnaeans: 44 nodes.
Ignatius to Polycarp: 32 nodes.
The Martyrdom of Polycarp: 85 nodes.
Polycarp to the Philippians: 54 nodes.
30947 total nodes on the tree.
python manage.py prepare_db  48.03s user 26.32s system 77% cpu 1:35.94 total
```

On `spike/speedy-ingestion`
```
time python manage.py prepare_db
<snip>
--[Loading versions]--
Iliad: 15710 nodes.
Odyssey: 12132 nodes.
Crito: 483 nodes.
Enchiridion: 157 nodes.
The Epistle to Diognetus: 113 nodes.
The Epistle of Barnabas: 218 nodes.
The First Epistle of Clement: 469 nodes.
The Second Epistle of Clement: 139 nodes.
The Didache: 119 nodes.
The Shepherd of Hermas: 923 nodes.
Ignatius to the Ephesians: 71 nodes.
Ignatius to the Magnesians: 43 nodes.
Ignatius to the Trallians: 45 nodes.
Ignatius to the Romans: 41 nodes.
Ignatius to the Philadelphians: 37 nodes.
Ignatius to the Smyrnaeans: 44 nodes.
Ignatius to Polycarp: 32 nodes.
The Martyrdom of Polycarp: 85 nodes.
Polycarp to the Philippians: 54 nodes.
30947 total nodes on the tree.
python manage.py prepare_db  5.04s user 0.33s system 90% cpu 5.935 total
```
_Refs  [Improve ingestion speed](https://trello.com/c/llhOXBlH)_

TODOS:
- [ ] Fix tests
- [ ] Improve internal documentation on the bulk_create flow

### CODE REVIEW

**Comment by jacobwegner** [comment]
Deployed to  https://mini-stack-a-spike-spee-wkno11.herokuapp.com/graphql/

**Comment by jacobwegner** [comment]
[Improve ingestion speed](https://trello.com/c/llhOXBlH/38-improve-ingestion-speed)

File: sv_mini_atlas/tests/test_importer.py [hunk 1]
```diff
@@ -243,169 +233,150 @@ def test_importer(mock_node, mock_open):
...
+        mock.call(
+            2,
+            {"kind": "textgroup", "urn": "urn:cts:greekLit:tlg0012:", "idx": 0},
+            "urn:cts:greekLit:",
+        ),
+        mock.call(
+            3,
+            {"kind": "work", "urn": "urn:cts:greekLit:tlg0012.tlg001:", "idx": 0},
+            "urn:cts:greekLit:tlg0012:",
         ),
```
Comment (jacobwegner) [inline sv_mini_atlas/tests/test_importer.py:262]: nice; this is a lot easier to reason about!

**Comment by jacobwegner** [comment]
@jhrr thanks for your work here; I want to add some docstrings but then will flag for a final review.

I just pushed a small change set where I removed some dead code and added simple assertions for `numchild` counts.

File: sv_mini_atlas/tests/test_importer.py (hunk 1 above)
Comment (jhrr) [reply sv_mini_atlas/tests/test_importer.py:262]: Yes, I was pleased about that too.

**Pull request merged**